### DISCUSSION TITLE
Tokens disappeared using Wormhole (SOL to ETH)

### DISCUSSION THREAD

**Discussion by ghost** [wormhole-foundation/wormhole#357] [Problems / bugs] [answered]
i tried to send wFTT from Sol to ETH ... the first part of the transaction went through and then it asked to confirm in metamask. It stated that the gas fees would be $1500 which was pretty high relative to the amount of tokens so i rejected it. After that the tokens were gone. Under recent activity on my Phantom Wallet it says success but the tokens are gone.

**Comment by oirman** [comment]
I have a similar story only transferring SPI.  I hope we get a solution.

**Reply by ghost** [reply]
i hope so as well. i thought i could reject the gas fees and try again later but i guess not.

**Comment by leoluk** [comment, marked as answer]
Needs tx hashes

**Reply by ghost** [reply]
5RuA7Zp1zPyrWLPjTTw4L4WazyJgQs6jQvkP8zzmvDESCSzNZA4aaGz6ZuTZGULJYQY5uLvEf8KE5F8MaqfVSUQa 

This is signature of the wormhole transaction.

**Reply by leoluk** [reply]
The transaction was successfully processed by the Wormhole network, but wormholebridge.com may have failed to submit the transaction. Try clearing your cache and reloading wormholebridge.com to see if the transaction can be completed there. Make sure both wallets are connected.

If wormholebridge.com doesn't work, you can manually submit the signed VAA bytes below to the submitVAA method on the Wormhole contract: https://etherscan.io/address/0xf92cd566ea4864356c5491c177a430c222d7e678#writeContract

If you need further help, please create a ticket with the wormholebridge.com project: https://github.com/solana-labs/oyster/issues

---

Digest: adba0609090755f94d3dd788af0c95fbc01772e1ec7c4995cf59dc9781a141d8

Signed VAA bytes:

> 0x01000000010d00d5709faf9085dbcdb3c9508355785d3d771edae5623c85e8660c51174fa5473247da0fc8ce2a9feda34bf55370d924c0b2cf225139b94aa4c8e8a8743136d15100018c2ffdff68f1fff44eb4308148bd9453892ea76fe30145732f113da639bf684b428462a94c7d1b27c31f1e67525deb3a2f04b1141194e18ebb3ff1dee94ffdca00048a5daeed4f5a42ba27ad005f8572633a8ce936799413704437c6bb7a3238acf87e1fa9fd5bebdb0b9a2fbcaea2f5899f4918113314c7ed9f78a8bbd9d533fa0101052244214836dea8f4fb2cd1e8dd4b63099aefc192bbb8d4e1098792c15ba25a26733fc0875deb7fd234bb04ba5bbe68752bfd49ccb108b024493823ce9304a4440107c3d5ed0697d4675880482312f601777f9cdceb29b1338bb51f7e86657973274337d95659853e6c1c869ce5e284284e39fe15bc887b43ef271d4641d33e615bd70008148a5b708d8454b1bf5784d45f38dee5a63493090491ba4482c757ec76daa39f072f967d43f0443f06604a39b108193fb3f80b3232973448323ae07dfd3936420109b6737dfda18136929814a6b8828600d30e73a34a8bc4763051516f81111b2158413fc9cf2207f57e2e6df06f737f70e21a073d0f55696d6d050a54e80bf45213000a06234ee7382f8cfc6a1c2ae0d65b8cbe874f81ae41718273279da1e8fa14c55d6cf5e6523448ae7cf42a7e10e59510e0cb9d93e3f5c97e8dd36ad0768e1d2692000bded9819dd7605c0cff1050d51760c5ffab9ec7746e1c21226d66b86778f3b5cc6b19c196a32f684282a6638dbe79f5ab3d6d307290a058dd5e3eb5f801c054cd010da2c70993abaf2a4adfc39625db2d2664b01c1269c4814cde32e7de804f57563d7d253021bd9cfc6d13680dd7c068c618df17607ebb0eee2f8510920e492d1445011005a54cea5fadcca7757bac79ca254dd3a22d2814032d22731d30124997666d340f09176074eb8f4867f8e50228c9e098d40d50476d338df43fc61a998f5eaee20011bac4777d14d476d57bee52e63b27de6f286f40153d7f12fd719226bb1a4a97ed6d7cb81830a87431ceeee24810995e74a91c74ec96696a403302201193b09af90012e48e3bd3cf1e9026e32092ca02fc92c8d2a717247f3f5a552f98388231b166be3596518aa14c1663287707c797dcf2007ff590dd93f3144f2a8257156a2aa6ee01612d38ed1000014a6e0102eaff924036f00aa4217ce09c81554975dac3a6113eeae1a74294f2e875b583c2000000000000000000000000e6375df92796f95394a276e0ba4efc4176d41d490200000000000000000000000050d1c9771902476076ecfc8b2a83ad6b9355a4c90900000000000000000000000000000000000000000000000000000021fe473400

**Reply by ghost** [reply]
Thanks. I see that this prompts the 2nd leg of the transaction in my ethereum wallet so that should get it done.

**Reply by leoluk** [reply]
Yep!

**Comment by KryptoKingz** [comment]
I just had a similar issue. My transaction was pending and it said there was a duplicate nonce which there wasn't. So I did a new transaction using the same nonce assuming it would cancel and spit the coins back to Solana. But not it seems the 1 BTC is really down a wormhole. I can't do the suggested action as the TX Hash doesn't exist as I overwrote the nonce... Pls help.
